exaudfclient/base/javacontainer/script_options/converter_v2.h

```diff
@@ -0,0 +1,41 @@
+#ifndef SCRIPTOPTIONLINEPARSERCONVERTERV2_H
+#define SCRIPTOPTIONLINEPARSERCONVERTERV2_H 1
+
+#include <string>
+#include <vector>
+#include <set>
+#include <memory>
+
+#include "base/javacontainer/script_options/converter.h"
+
+
+
+namespace SWIGVMContainers {
+
+namespace JavaScriptOptions {
+
+class ConverterV2 : public Converter {
+
+    public:
+        ConverterV2();
+    
+        void convertExternalJar(const std::string & value);
+
+        const std::set<std::string> & getJarPaths() const {
+            return m_jarPaths;
+        }
```

Comment:
Often, it is sufficient to expose only iterators rather than the full underlying (internal) data structure.

Reply:
This function will change in a future PR completely. I only wanted to keep the size of the PR as small as possible.